### ISSUE TITLE
How to use api.dom.compose or api.dom.composes to insert body and subject?

### ISSUE THREAD

**Issue by dragfire** [KartikTalwar/gmail.js#372]
I've tried these:
```
  gmail.observe.on("compose", function(obj) {
    var $composeElement = obj.$el;
    console.info('composeElement', $composeElement, $($composeElement).hasClass("M9"), $($composeElement).hasClass("AD"));
    var $compose = gmail.dom.compose($composeElement);
    $compose.subject("Hello there");
    $compose.body("Yes!!!");
  });
```

or 

```
 gmail.observe.on("compose", function(obj) {
    var $composeElement = gmail.dom.composes()[0];
    $composeElement.body("Hello");
    $composeElement.subject("There!!!");
    console.info('composeElement', $composeElement, $($composeElement).hasClass("M9"), $($composeElement).hasClass("AD"));
}
```

In source code https://github.com/KartikTalwar/gmail.js/blob/master/src/gmail.js#L2438
`$($composeElement).hasClass("AD")` is always `false`.

Can someone help me in solving this?

**UPDATE**
It's working if I add a `setTimeout`.
```
  gmail.observe.on("compose", function(obj, type) {
    console.info(obj, obj.$el, $(obj.$el));
    window.setTimeout(function () {
      gmail.dom.compose(obj.$el).body("Hello there");
      obj.subject("Hello There");
    }, 500);
  })
```

**Comment by josteink** [comment]
You just use the object directly: `obj` is a `gmail.dom.compose`-instance. You're right about needing a setTimeout though.

This should be as little as you need:

````javascript
gmail.observe.on("compose", function(obj) {
    setTimeout(() => {
        obj.subject("boo");
        obj.body("foo");
    }, 100);
});
````

TLDR: Don't go recreate DOM-objects when you already have it. Cheers.
